src/tags.cpp

```diff
@@ -19,7 +19,7 @@ namespace Exiv2 {
 using namespace Internal;
 
 //! List of all defined Exif sections.
-constexpr auto sectionInfo = std::array{
+constexpr std::array<SectionInfo, 22> sectionInfo{
```

Comment:
What's the error here?

Reply:
I'll check again tomorrow with the raspberry pi and copy&paste the error here.

Reply:
Strange ... I dropped the commit and everything compiles now correctly. I might have get confused between the output from the compiler and the errors appearing in an old version of QtCreator. 

I will drop this commit from the PR.